src/OpenRasta/Diagnostics/TraceSourceLogger.cs

```diff
@@ -4,6 +4,11 @@
 
 namespace OpenRasta.Diagnostics
 {
+  public static class DefaultLogger
+  {
+    public static ILogger Instance { get; set; } = new TraceSourceLogger();

```

Comment:
I did my best in each spot, but the intent would be to set the logger first-thing and never change it:
https://github.com/ololabs/platform/blob/5ef2f9ec5f615c0b40e1bfdaa48027014fdbe0a0/PlatformAPI/1.1/src/Program.cs#L20-L24

### Questionable

- Should this just use `DefaultLogger.Instance` instead of resolving? https://github.com/ololabs/openrasta-core/blob/7fbf478f1cd4af67d2f59f64ae450fb5f4e0ef0b/src/OpenRasta/Hosting/HostManager.cs#L24-L27 https://github.com/ololabs/openrasta-core/blob/7fbf478f1cd4af67d2f59f64ae450fb5f4e0ef0b/src/OpenRasta/Hosting/HostManager.cs#L98-L101
 
   <details><summary>That said, it's fine - the ctor is called via the `OpenRastaMiddleware` ctor.</summary>

    https://github.com/ololabs/openrasta-core/blob/eeb9f92bfd33c8b088705e5aacf1da3fa48febdc/src/OpenRasta/Hosting/Owin/OpenRastaMiddleware.cs#L63-L71
    https://github.com/ololabs/openrasta-core/blob/eeb9f92bfd33c8b088705e5aacf1da3fa48febdc/src/OpenRasta/Hosting/Owin/OpenRastaMiddleware.cs#L100-L108
    https://github.com/ololabs/openrasta-core/blob/eeb9f92bfd33c8b088705e5aacf1da3fa48febdc/src/OpenRasta/Hosting/HostManager.cs#L55-L59

    </details>

### Pass-Through

- https://github.com/ololabs/openrasta-core/blob/7fbf478f1cd4af67d2f59f64ae450fb5f4e0ef0b/src/OpenRasta/Pipeline/LoggingMiddleware.cs#L10
- https://github.com/ololabs/openrasta-core/blob/7fbf478f1cd4af67d2f59f64ae450fb5f4e0ef0b/src/OpenRasta/DI/Internal/ObjectBuilder.cs#L16
- https://github.com/ololabs/openrasta-core/blob/7fbf478f1cd4af67d2f59f64ae450fb5f4e0ef0b/src/OpenRasta/Pipeline/ThreePhasePipelineInitializer.cs#L14
- https://github.com/ololabs/openrasta-core/blob/7fbf478f1cd4af67d2f59f64ae450fb5f4e0ef0b/src/OpenRasta/Pipeline/Contributors/ResponseEntityWriterContributor.cs#L18

### Per Request

- https://github.com/ololabs/openrasta-core/blob/7fbf478f1cd4af67d2f59f64ae450fb5f4e0ef0b/src/OpenRasta/Hosting/Owin/OwinHost.cs#L60-L62

### Unused (`AspNet`)

- https://github.com/ololabs/openrasta-core/blob/7fbf478f1cd4af67d2f59f64ae450fb5f4e0ef0b/src/OpenRasta.Hosting.AspNet/AspNetCommunicationContext.cs#L23
- https://github.com/ololabs/openrasta-core/blob/7fbf478f1cd4af67d2f59f64ae450fb5f4e0ef0b/src/OpenRasta.Hosting.AspNet/AspNetResponse.cs#L23
- https://github.com/ololabs/openrasta-core/blob/7fbf478f1cd4af67d2f59f64ae450fb5f4e0ef0b/src/OpenRasta.Hosting.AspNet/OpenRastaModuleAsync.cs#L44